### PR TITLE
[SCC-4455] Support multiple 910|a fields

### PR DESCRIPTION
**Pull request by yossariano**
See https://newyorkpubliclibrary.atlassian.net/browse/SCC-4455 - this change adds support for checking multiple `910|a` values for whether a bib/item is meant for research, instead of just the first value in the field.

#### Testing
* Updated tests to include a fixture with both `BL` and `RL` in the `910|a` field.

### CODE REVIEW

File: lib/sierra-models/bib.js [hunk 1]
```diff
@@ -103,14 +103,14 @@ class SierraBib extends SierraBase {
...
 
   /**
-   *  Get the value in 910 $a if one exists.
+   *  Get the values in 910 $a.
    *
-   *  This value typically has RL for Research bibs, BL for Branch bibs, or
+   *  These values typically are RL for Research bibs, BL for Branch bibs, or
    *  RLOTF for OTF (Research) bibs
    */
   _varField910 () {
```
Comment (charmingduchess) [inline lib/sierra-models/bib.js:111]: this is not technically in scope, but could you update this method name to something like `_varField910a`?

File: lib/sierra-models/bib.js [hunk 2]
```diff
@@ -70,10 +70,10 @@ class SierraBib extends SierraBase {
     }
 
     // Check 910 $a for RL or BL:
-    const researchBranchFlag = this._varField910()
-    if (researchBranchFlag) {
-      const isResearch = ['RL', 'RLOTF'].includes(researchBranchFlag)
-      return { isResearch, rationale: `910 $a is ${researchBranchFlag}` }
+    const researchBranchFlags = this._varField910()
+    if (researchBranchFlags.length !== 0) {
+      const isResearch = researchBranchFlags.includes('RL') || researchBranchFlags.includes('RLOTF')
```
Comment (charmingduchess) [inline lib/sierra-models/bib.js:75]: is this just a stylistic preference?

**Review by charmingduchess — CHANGES_REQUESTED** [review]

**Comment by charmingduchess** [comment]
I'm also not seeing any prior unit tests for this method. I think it would be a good idea to add some edge case coverage for `isOtfRecord` or maybe updating [these suppression tests](https://github.com/NYPL/research-catalog-indexer/blob/SCC-4455-multiple-910a/test/unit/sierra-bib.test.js).

File: lib/sierra-models/bib.js (hunk 2 above)
Comment (yossariano) [reply lib/sierra-models/bib.js:75]: Yeah, I'm open to any style suggestions!

File: lib/sierra-models/bib.js (hunk 1 above)
Comment (yossariano) [reply lib/sierra-models/bib.js:111]: Absolutely, will update.

**Comment by yossariano** [comment]
> I'm also not seeing any prior unit tests for this method. I think it would be a good idea to add some edge case coverage for `isOtfRecord` or maybe updating [these suppression tests](https://github.com/NYPL/research-catalog-indexer/blob/SCC-4455-multiple-910a/test/unit/sierra-bib.test.js).

It is implicitly tested with this test (it failed with multiple `910|a` fields previously):

```
  prefilter
    filteredSierraBibsForBibs
      1) should keep Research bibs
```

but I'm adding more explicit testing now!

**Review by nonword — APPROVED** [review]
